tools/ansible/roles/dockerfile/templates/Dockerfile.j2

```diff
@@ -274,6 +274,8 @@ RUN for dir in \
       /var/run/nginx.pid \
       /var/lib/awx/venv/awx/lib/python3.9/site-packages/awx.egg-link ; \
     do touch $file ; chmod g+rw $file ; done
+
+RUN echo "\setenv PAGER 'less -S'" > /var/lib/awx/.psqlrc
```

Comment:
Maybe combine with the `RUN` right above it, just to avoid another image layer?

Reply:
like this?

Reply:
Yep, lgtm